### PR TITLE
fix: duckling entities and CRFExctractor

### PR DESCRIPTION
**Pull request by MatthieuJnon**

### CODE REVIEW

**Comment by pheel** [comment]
Isn' it what `rasa_addons.nlu.components.duckling_crf_merger.DucklingCrfMerger` does?

File: botfront/imports/api/graphql/common/common.types.graphql [hunk 1]
```diff
@@ -47,6 +47,7 @@ input EntityInput {
     start: Int
     end: Int
     confidence: Float
+    confidence_entity: Float
```
Comment (kenrilea) [inline botfront/imports/api/graphql/common/common.types.graphql:50]: could I get a quick explanation on what this is for? I can't find it referenced anywhere else.

**Review by kenrilea — COMMENTED** [review]
--

File: botfront/imports/api/graphql/common/common.types.graphql (hunk 1 above)
Comment (MatthieuJnon) [reply botfront/imports/api/graphql/common/common.types.graphql:50]: WHen I use crfEntityExtractor that's what confidence is named, and Graphql doesn't save if there are unknown properties

**Review by kenrilea — CHANGES_REQUESTED** [review]
examples with duckling entities can be added to NLU examples if typed in the webchat and then added to examples via incoming. When an example with a duckling entity is present training fails.

**Review by kenrilea — APPROVED** [review]

**Pull request merged**